kubernetes/samples/charts/voyager/README.md

```diff
@@ -118,18 +118,18 @@ $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:30305/testwebapp/
 $ curl -k -H 'host: domain2.org' https://${HOSTNAME}:30307/testwebapp/
 ```
 
-## SSL Termination at Ingress Controller
-This sample demonstrates how to terminate SSL trafik at ingress contoller to access WebLogic console thru SSL port. 
+## SSL termination at ingress controller
+This sample demonstrates how to terminate SSL traffic at ingress controller to access the WebLogic Server Administration Console through SSL port. 
```

Comment:
at ingress controller -> at the ingress controller
through SSL port-> through the SSL port

Reply:
done